src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int length;
+    private int width;
+
+    public Rectangle(String color, int length, int width) {
+        setColor(color);
+        this.length = length;
+        this.width = width;
+    }
+
+    @Override
+    public double calculateArea() {
+        return this.length * this.width;
```

Comment:
No need to use "this" here, it is redundant. Also you can remove "this" in other figures methods "calculateArea" and "toString"
```suggestion
        return length * width;
```